mirrord-layer/src/lib.rs

```diff
@@ -425,11 +425,40 @@ fn enable_hooks() {
         )
         .unwrap();
 
-    interceptor
-        .replace(
-            Module::find_export_by_name(None, "accept").unwrap(),
-            NativePointer(accept_detour as *mut c_void),
-            NativePointer(std::ptr::null_mut::<c_void>()),
-        )
-        .unwrap();
+    match Module::find_export_by_name(None, "uv__accept4") {
```

Comment:
change to if let some, and maybe create a hashset of already hooked addresses so we won't hook twice?
also, just to be on the safe side, create a detour for accept4 because it accepts 1 more argument.